api/transport_windows.go

```diff
@@ -0,0 +1,17 @@
+package api
```

Comment:
Actually, looking back at this. The npipe package is only for Windows (only builds for Windows), which makes sense as it's a Windows feature. So we'd have to differentiate after all.